central/reports/scheduler/schedule.go

```diff
@@ -310,25 +322,28 @@ func (s *scheduler) sendReportResults(req *ReportRequest) error {
 	return nil
 }
 
-func formatMessage(rc *storage.ReportConfiguration) (string, error) {
+func getBrandedProductName() string {
+	if branding.GetProductBrandingEnvName() == "RHACS_BRANDING" {
+		return productBrandingNameRHACS
+	}
+	return productBrandingNameStackrox
+}
+
+func formatMessage(rc *storage.ReportConfiguration, EmailTemplate string) (string, error) {
```

Comment:
[nit] I don't think that `EmailTemplate` variable name is Go style. It should be `emailTemplate`.

Reply:
Good point, strange that make style / quickstyle didn't catch it since I ran both.